README.md

```diff
@@ -1,50 +1,65 @@
 # roboteam_suite
-All relevant roboteamtwente repositories in one.
+This repo contains:
+- Roboteam AI, which listens to Roboteam World and makes decisions on what the robots should do based on the vision data.
```

Comment:
And robothub

Reply:
Hub receives feedback but I'm not sure if that's redirected to AI directly or goes through world first

Reply:
directly to Ai, we're planning to change this.